examples/artifact_benchmarking/src/benchmark.py

```diff
@@ -60,8 +59,10 @@ def precision(model: Module, test_dataloader: DataLoader, padding_id: int = -100
         {"class_label": "B-MISC"},
         {"class_label": "I-MISC"},
     ),
-    tags=("metric", "per-class"),
-)
+    tags=("metric", "per-class"))()
```

Comment:
The last call `()` was missing in previous implementations and was what broke the collection in the version previously.